rackcity/views/it_instance_views.py

```diff
@@ -214,6 +214,13 @@ def instance_modify(request):
             value = ITModel.objects.get(id=data[field])
         elif field == 'rack':
             value = Rack.objects.get(id=data[field])
+        elif field == 'hostname':
+            if len(ITInstance.objects.filter(hostname__iexact=data[field])) != 0:
+                return JsonResponse(
+                    {"failure_message": "Instance with hostname '" +
+                        data[field].lower() + "' already exists."},
+                    status=HTTPStatus.BAD_REQUEST,
+                )
```

Comment:
I'm confused by this part - if you're modifying an instance by changing the case of the hostname (I thought that was allowed) wouldn't this stop you from doing it?